cleverhans/attacks.py

```diff
@@ -308,14 +328,25 @@ def generate(self, x, **kwargs):
         # Fix labels to the first model predictions for loss computation
         model_preds = self.model.get_probs(x)
         preds_max = tf.reduce_max(model_preds, 1, keep_dims=True)
-        y = tf.to_float(tf.equal(model_preds, preds_max))
```

Comment:
This was actually a bug in the old code -- it would never read the self.y params.